packages/postgraphile-core/__tests__/kitchen-sink-schema.sql

```diff
@@ -433,6 +437,8 @@ begin
 end;
 $$ language plpgsql stable;
 
+comment on function c.badly_behaved_function() is '@deprecated This is deprecated.';
+
```

Comment:
In future, putting different comments on each thing makes it easier to confirm that the right things are being deprecated for the right reasons. `This is deprecated (badly_behaved_function).` would have been safer.